modules/administration/pages/ssl-certs-hsts.adoc

```diff
@@ -5,15 +5,21 @@ HTTP Strict Transport Security (https://developer.mozilla.org/en-US/docs/Web/HTT
 
 {productname} allows enabling HSTS, to enable it for a {productname} Server:
 
+.Procedure
```

Comment:
For formal procedure, we probable better avoid the 1., 2., etc. numbering.

Reply:
They were from the beginning. I am not changing anything structurally in the file, just the formatting for the purpose of consistency.